bigchaindb/models.py

```diff
@@ -165,8 +165,6 @@ class Block(object):
         node_pubkey (str): The public key of the node creating the
             Block.
         timestamp (str): The Unix time a Block was created.
-        voters (:obj:`list` of :obj:`str`): A list of a federation
```

Comment:
Since you are removing `voters` from the docs, makes sense to remove them from the method as well.

Reply:
There is a lot of voting-related code that can be removed but most of that can probably be deferred to another pull request, otherwise this pull request will get enormous. It's okay to remove some in this PR; let's just not get carried away :D